doc/usecases/kdb/UC_validate_config.md

```diff
@@ -0,0 +1,37 @@
+# Use Case: Ensuring valid configuration
```

Comment:
I think this is simply a post-condition of "Modifying application configuration".

Reply:
Then the name is misleading. So you mean something like:

```suggestion
# Use Case: Configuration with specification
```